### DISCUSSION TITLE
Is it possible to implement long screenshot?

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2631] [Q&A] [answered]
In my testing workflow, I often need to capture full-page screenshots to ensure that all elements on the page are correctly displayed and rendered. However, the default behavior of self.save_screenshot()  captures only the visible portion of the page within the dimension of the screen.

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/discussions/902#discussioncomment-723429

If you run tests using `--headless` mode, you'll get the full page screenshot.
Otherwise they will be the same size as the browser window.

Also try [examples/test_image_saving.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_image_saving.py)
It shows you how to take screenshots of specific sections of web pages.
